app.js

```diff
@@ -5,11 +5,45 @@ app.use(express.json());
 
 const people = [];
 
+// Routes
+//post - add entry
+app.post('/api/v1/people', (req, res) => {
+  const person = { 
+    name: req.body.name,
+    age: req.body.age
+  }
+  if(!person.name) {
+    return res.status(400).json({error:'Please enter a name.'});
+  } else if (!person.age) {
+    return res.status(400).json({error:'Please enter an age.'});
+  }
+  let length = people.push(req.body);
+  res.json({msg:'A person record was added', index: length-1});
+});
+
+//get the array
+app.get('/api/v1/people', (req, res) => {
+  res.json({ people });
+});
 
-app.all("/api/v1/*", (req, res) => {
-  res.json({ error: "That route is not implemented." });
+//get single entry from array
+app.get('/api/v1/people/:id', (req, res) => {
+  let index = req.params.id;
+  index = Number(index);
+  let person = people[index];
+  if(!person) {
```

Comment:
I think you need an if/else here, otherwise when the person is not found it will send a 404 status code and then also try to send `res.json({ person });`